p4-fusion/branch_set.cc

```diff
@@ -7,13 +7,15 @@
 #include "branch_set.h"
 #include "minitrace.h"
 #include <map>
+#include <utility>
 
-static const std::string EMPTY_STRING = "";
+static const std::string EMPTY_STRING;
 static const std::array<std::string, 2> INVALID_BRANCH_PATH { EMPTY_STRING, EMPTY_STRING };
 
 std::vector<std::string> BranchedFileGroup::GetRelativeFileNames()
 {
 	std::vector<std::string> ret;
+	ret.reserve(files.size());
```

Comment:
This function is unused, you can delete the definition and the corresponding declaration in the header.

More generally, it's generally a design smell in C++ if you're copying lots of strings around, because those are deep copies, not shallow copies unlike Go or Java.